docs/developer-guide/create-control-panel.md

```diff
@@ -0,0 +1,379 @@
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:i18n="http://namespaces.zope.org/i18n"
+    i18n_domain="mypackage">
+    
+    <!-- Other configuration -->
+    
+    <include package=".controlpanel" />
+    
+</configure>
+```
+
+### Add the control panel entry
+
+Create a {file}`mypackage/profiles/default/controlpanel.xml` in your package's GenericSetup profile with the following content to add your control panel to the Plone control panel listing.
+
+```xml
+<!-- mypackage/profiles/default/controlpanel.xml -->
+<?xml version="1.0"?>
+<object name="portal_controlpanel">
+    <configlet
+        title="My Addon Settings"
+        action_id="my-controlpanel"
+        appId="my.addon"
+        category="plone-general"
+        condition_expr=""
+        icon_expr="string:puzzle"
+        url_expr="string:${portal_url}/@@my-controlpanel"
+        visible="True">
+        <permission>Manage portal</permission>
+    </configlet>
+</object>
+```
+
+The category attribute can be one of the following values.
+These values correspond to the groups in Site Setup.
+
+`plone-general`
+:   General settings
+
+`plone-content`
+:   Content-related settings
+
+`plone-users`
+:   Users and groups settings
+
+`plone-security`
+:   Security settings
+
+`plone-advanced`
+:   Advanced settings
+
+
+### Set default values in the registry
+
+Define default values for your settings in {file}`mypackage/profiles/default/registry.xml`.
+
+```xml
+<!-- mypackage/profiles/default/registry.xml -->
+<?xml version="1.0"?>
+<registry>
+    <records interface="mypackage.controlpanel.settings.IMyControlPanelSettings"
+             prefix="my.addon">
+        <value key="my_setting">default value</value>
+        <value key="my_choice">value3</value>
+    </records>
+</registry>
+```
+
+
+### Access your settings in code
+
+You can access your settings in Python code as follows.
+
+```python
+from plone.registry.interfaces import IRegistry
+from zope.component import getUtility
+
+registry = getUtility(IRegistry)
+settings = registry.forInterface(IMyControlPanelSettings, prefix="my.addon")
+
+# Now you can access the settings
+my_setting_value = settings.my_setting
+my_choice_value = settings.my_choice
+```
+
+
+## Register a control panel
+
+To manually register a view as a control panel, add the following registration to your {file}`/profiles/default/controlpanel.xml`.
+
+```xml
+<?xml version="1.0"?>
+  <object
+      name="portal_control-panel"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      i18n:domain="lmu.behavior">
+    <configlet
+      title="Some Control Panel"
+      action_id="collective.example.some_control-panel"
+      appId="collective.example"
+      category="Products"
+      condition_expr=""
+      url_expr="string:${portal_url}/@@some_view"
+      icon_expr=""
+      visible="True"
+      i18n:attributes="title">
+          <permission>Manage portal</permission>
+    </configlet>
+  </object>
+```
+
+
+## Use `FieldSet` to group fields
```

Comment:
Is this also part of the manual process only or both manual and plonecli?

Reply:
@stevepiercy this needs to be added in any way you use to create the controlpanel, as these include the fields (to be grouped) that needs to be shown in the settings (of the controlpanel)

Reply:
OK, please see other comments regarding the flow and structure of this chapter. I think we're close to finished with those enhancements.